src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -36,6 +36,9 @@ public class GitHubClient : RemoteRepoBase, IGitRepo
         private static readonly Regex PullRequestUriPattern =
             new Regex(@"^/repos/(?<owner>[^/]+)/(?<repo>[^/]+)/pulls/(?<id>\d+)$");
 
+        private static readonly Regex DependencyUpdatesPattern =
+            new Regex(@"- \*\*(?<update>\w+)\*\*: from (?<oldVersion>[\d\.]*) to (?<newVersion>[\d\.]+)");
+
```

Comment:
Should add an example here to show what this regex actually captures?

Reply:
yes, any regex should have examples with it.